src/modules/python/libkvipython.cpp

```diff
@@ -502,39 +502,13 @@ static bool python_kvs_fnc_isAvailable(KviKvsModuleFunctionCall * c)
 	return true;
 }
 
-/*
-	@doc: python.version
-	@type:
-		function
-	@title:
-		$python.version
-	@short:
-		Check which version of Python is supported in this build of KVIrc
-	@syntax:
-		$python.version
-	@description:
-		Returns which major version of Python is KVIrc linked to ([b]2[/b] or [b]3[/b])
-		or [b]0[/b] if Python is not supported at all.
-*/
-
-static bool python_kvs_fnc_version(KviKvsModuleFunctionCall * c)
-{
-#ifdef COMPILE_PYTHON_SUPPORT
-	c->returnValue()->setInteger(PY_MAJOR_VERSION);
-#else
-	c->returnValue()->setBoolean(false);
-#endif
-	return true;
-}
-
 static bool python_module_init(KviModule * m)
 {
 	// register the command anyway
 	KVSM_REGISTER_SIMPLE_COMMAND(m, "begin", python_kvs_cmd_begin);
 	KVSM_REGISTER_SIMPLE_COMMAND(m, "destroy", python_kvs_cmd_destroy);
 
 	KVSM_REGISTER_FUNCTION(m, "isAvailable", python_kvs_fnc_isAvailable);
-	KVSM_REGISTER_FUNCTION(m, "version", python_kvs_fnc_version);
```

Comment:
This seems like a potentially useful information to expose to consumers of this API.
Perhaps it should be called `python_version` instead of `version`.

Reply:
I withdraw my suggestions.
Something like the following already works:
```
python.begin
import sys
kvirc.setLocal("python_major_version", str(sys.version_info[0]))
kvirc.setLocal("python_minor_version", str(sys.version_info[1]))
kvirc.setLocal("python_micro_version", str(sys.version_info[2]))
python.end
echo %python_major_version
echo %python_minor_version
echo %python_micro_version
```